custom/enikshay/tasks.py

```diff
@@ -127,10 +149,24 @@ def update_single_case(self, episode_case):
             update_case(self.domain, episode_case.case_id, update_json)
 
     def _get_open_episode_cases(self):
-        # return all open 'episode' cases
         case_accessor = CaseAccessors(self.domain)
         case_ids = case_accessor.get_open_case_ids_in_domain_by_type(CASE_TYPE_EPISODE)
-        return case_accessor.iter_cases(case_ids)
+        episode_cases = case_accessor.iter_cases(case_ids)
+
+        for episode_case in episode_cases:
+            # if this episode is part of a deleted or archived person, don't update
+            try:
+                person_case = get_person_case_from_episode(self.domain, episode_case.case_id)
+            except ENikshayCaseNotFound:
+                continue
```

Comment:
This happens when the person case was deleted during a migration event. At some point, we'll probably want to close / delete all the episodes that fall into this category, but will leave that as a `TODO`, as that process cascades pretty heavily (since there are __many__ related extension cases that are attached to an episode)